fe/src/pages/Home/HomeContext.ts

```diff
@@ -4,11 +4,15 @@ import { ViewType } from '../../utils';
 interface HomeContextType {
   view: ViewType;
   setView: React.Dispatch<React.SetStateAction<ViewType>>;
+  page: number;
```

Comment:
Usually the context state is under a single type.
In this case that would mean that we would have a `HomeState` which has `page` and `view` in it.  I guess this works as well, just food for thought.

Reply:
> Overall nice, but I think some people might have a hard time understanding what's going on. I think just adding the classic `...` between the first and last elements in pagination will help.

I could try adding that, problem might be trying to fit it to a mobile screen, currently it's right on the edge. But i'll try to figure something out to fit it.

Reply:
I think on a mobile screen you won't be able to have it in the same row with the view select buttons, so just push it to the next row and then you'll have more space

Reply:
@Orzelius It turned out to be quite complicated to simply add some dots for the paginator, i didn't want to spend too much time on it. I updated the styling though, for easier "Back" and "Next" movement, etc, perhaps this is an improvement? Hopefully its good enough for now. 😅 

New demo:


https://user-images.githubusercontent.com/48998862/119118949-62e1dd80-ba33-11eb-9514-e9427c03f504.mp4